.pep8speaks.yml

```diff
@@ -0,0 +1,2 @@
+pycodestyle:
+  max-line-length: 88  # To be compatible with black
```

Comment:
curious: why particularly 88, and what's black?

Reply:
Well, generally the community answer to make the PEP8 code style was to use a lint tool like `flake8` or `pycodestyle` and etc. These tools only report the problem, do not fix them. On the other hand, [`black`](https://github.com/psf/black) is an auto-code-formatter (as they say).

You just run it and it automatically fixes the problem. There are also some Jupyter extensions, such as [this one](https://github.com/drillan/jupyter-black), that run `black` inside Jupyter notebook.

And by the way, `black` uses 88, [by default](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length), characters per line.

You can read more about it on their [documentation site](https://black.readthedocs.io/en/stable/#). It's a great tool and I hope you’ll check it out!

Reply:
black is awesome. Look at https://www.thoughtworks.com/radar/techniques/opinionated-and-automated-code-formatting and https://www.youtube.com/watch?v=esZLCuWs_2Y